### ISSUE TITLE
fwbuilder crashes on iptables import

### ISSUE THREAD

**Issue by computersalat** [fwbuilder/fwbuilder#9]
admin@lap:~/security/fwBuild> fwbuilder 
Firewall Builder GUI 5.3.6
platform= "iptables" 
fwbuilder: ../libfwbuilder/src/fwbuilder/Dispatch.h:180: virtual void\* libfwbuilder::Dispatch::dispatch(libfwbuilder::Interface_, void_): Zusicherung »"Call to unimplemented virtual void\* Dispatch::dispatch(""Interface""_, void_)"==__null« nicht erfüllt.
Abgebrochen

**Comment by sirius** [comment]
If you can trim a iptables config to the bare minimum - that would still trigger the crash - and send it to me, it will be very helpful :)

**Comment by cwittmer** [comment]
Hi Sirius,
sent you a private Mail with my iptables-save ... hope you can find the problem ...

**Comment by sirius** [comment]
@cwittmer Did you send to sirius@bakke.co?

**Comment by cwittmer** [comment]
Am 05.09.2016 um 23:43 schrieb Sirius Bakke:
> @cwittmer <https://github.com/cwittmer> Did you send to sirius@bakke.co
> <mailto:sirius@bakke.co>?
>  
yes I did

-- 

Christian
----------------------------------------------------
   - Please do not 'CC' me on list mails.
          Just reply to the list :)
----------------------------------------------------
Der ultimative shop für Sportbekleidung und Zubehör

http://www.sc24.de
----------------------------------------------------

**Comment by cwittmer** [comment]
sent again an answer to your last 'personal' mail ...

**Comment by sirius** [comment]
@cwittmer Thanks, I found it.

**Comment by sirius** [comment]
This crash is introduced by the DummyInterface feature from 1e685abd1f5282a8b6c4a159831cc498cec14a60.

@cwittmer Can you confirm that unchecking "Find and use existing objects" is preventing the crash?

**Comment by cwittmer** [comment]
Yes, this is preventing from crashing ... but then running into Errors 
iptables_sys-02_20160904.save:
```
1: Version: 1.4.21
3: New ruleset: nat / PREROUTING
3: Default action: Accept
4: New ruleset: nat / INPUT
4: Default action: Accept
5: New ruleset: nat / OUTPUT
5: Default action: Accept
6: New ruleset: nat / POSTROUTING
6: Default action: Accept
7: New interface: eth0
8: New interface: eth1
13: Version: 1.4.21
13: Error: Unrecognized netfilter table "raw". Only tables "filter", "mangle" and "nat" are supported.
15: New ruleset: raw / PREROUTING
15: Default action: Accept
16: New ruleset: raw / OUTPUT
16: Default action: Accept
Parser error: line 17:30: expecting ''a'', found ''o''
17: Warning: Line 16: Added rule to reproduce default policy ACCEPT in raw/OUTPUT
17: Warning: Line 15: Added rule to reproduce default policy ACCEPT in raw/PREROUTING
```
ip6tables_sys-02_20160904.save:
```
Parser error: line 1:18: expecting ''t'', found ''6''
Parser error:
Could not find enough information in the data file to create firewall object.


Please check that the file you are trying to import is in one of supported formats. Currently fwbuilder can only import iptables configuration saved with 'iptables-restore' command, Cisco routers (IOS), Cisco ASA, FWSM and PIX configurations saved with 'show run' command and PF configuration from a pf.conf file.
```
not ready for IPv6 import with ip6tables-restore ?

**Comment by sirius** [comment]
Fixed in 95072938d16b93935c5941b86d0334ca379f9b5d

The import feature seems to be a bit outdated. If you need it, please open a new issue.
